src/main/java/com/fasterxml/jackson/databind/deser/std/StdKeyDeserializer.java

```diff
@@ -410,9 +410,11 @@ private EnumResolver _getToStringResolver(DeserializationContext ctxt)
             EnumResolver res = _byToStringResolver;
             if (res == null) {
                 synchronized (this) {
-                    res = EnumResolver.constructUsingToString(ctxt.getConfig(),
-                            _byNameResolver.getEnumClass());
-                    _byToStringResolver = res;
+                    if (res == null){
```

Comment:
nit: could you add a space before the `{`

Reply:
Done

Reply:
I really do need to get my editor settings fixed!
Thanks for being patient ✌️✌️